application/common/models/User.php

```diff
@@ -145,7 +151,7 @@ private function savePassword($runValidation = true): bool
 
             $this->password_hash = password_hash($this->password, PASSWORD_DEFAULT);
 
-            if (! parent::save($runValidation, ['password_hash'])) {
+            if (! parent::save()) {
```

Comment:
yes, I can understand that..., sorry.  I meant no, it was not necessary.  This continues to go back to the idea of the "active" attributes involved in the scenario.  There's no way for any other attributes to make it into the mix.  It's certainly a worthwhile test to have though so I'm putting a reminder in place for it.  https://github.com/silinternational/idp-id-broker/commit/b794df9a4b8a235de5405e4f63980cc189843027